pkg/errdefs/helper.go

```diff
@@ -0,0 +1,20 @@
+package errdefs
+
+type errNotFound struct{ error }
+
+func (errNotFound) NotFound() {}
+
+func (e errNotFound) Cause() error {
+	return e.error
+}
+
+func (e errNotFound) Unwrap() error {
```

Comment:
when do we need this?

Reply:
It is a convention of the go community that structs that encapsulate errors are generally provided with an Unwrap to return the original error.

We don't currently use